Resources/ArmTemplates/parameters.json

```diff
@@ -57,16 +42,16 @@
         "value": "__cosmosDbCollectionApprenticeshipReport__"
       },
       "providerServiceSettingsApiUrl": {
-        "value": "__providerServiceSettingsApiUrl__"
+        "value": "__providerApiUrl__"
       },
       "providerServiceSettingsApiKey": {
-        "value": "__providerServiceSettingsApiKey__"
+        "value": "__ApimPublishCourseDirectoryKey__"
```

Comment:
This is defined in a shared variable group, and currently is set to the Apim key of the FAC api until the end of the migration process, where it will get set to the correct key